prov/rxm/src/rxm.h

```diff
@@ -724,31 +724,28 @@ rxm_ep_normal_send(struct rxm_ep *rxm_ep, struct rxm_conn *rxm_conn,
 static inline
 struct rxm_buf *rxm_buf_get(struct rxm_buf_pool *pool)
 {
-	struct rxm_buf *buf;
-
-	pool->rxm_ep->res_fastlock_acquire(&pool->lock);
-	buf = util_buf_alloc(pool->pool);
-	pool->rxm_ep->res_fastlock_release(&pool->lock);
-	return buf;
+	return util_buf_alloc(pool->pool);
 }
 
 static inline
 void rxm_buf_release(struct rxm_buf_pool *pool, struct rxm_buf *buf)
 {
-	pool->rxm_ep->res_fastlock_acquire(&pool->lock);
 	util_buf_release(pool->pool, buf);
-	pool->rxm_ep->res_fastlock_release(&pool->lock);
```

Comment:
It's not worth keeping rxm_buf_get/release without the locks, since they are 1 line functions.

Reply:
I fixed your comment by applying @a-ilango comment regarding two functions.